Kernel/Output/HTML/Preferences/Password.pm

```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 maxence business consulting GmbH, http://www.maxence.de
```

Comment:
We have changed our rules regarding copyright.
We therefore ask you to remove your newly added copyrights.

```
Znuny is an open fork of an existing software. 
So we have to respect the already given copyright of the original creators.

New files will be licensed using the AGPL Version 3. 
If you contribute code to the Znuny project you will get mentioned in the pull request 
incl. the commit, in CHANGES.md and in AUTHORS.md. 
We will not mention you in the file you provided or changed. 
Your work is highly appreciated and acknowledged 
but you contribute it to the project and your copyright will pass on to the fork itself.
```

https://github.com/znuny/Znuny/blob/rel-6_1-dev/CONTRIBUTING.md#licensing-copyright-and-credits


```suggestion
```